weave/guides/platform/weave-projects.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Weave projects overview"
+description: "Weave projects are collections of traces, evaluations, models, and other Weave assets"
```

Comment:
How about something like "Use Weave projects to organize related assets like traces, prompts, evaluations, models, and dashboards"? FYI the description also shows up at the top of the page content, so you may not need to duplicate it all in the first paragraph, depending on what you think about the preview.